packages/@okta/vuepress-site/docs/guides/request-user-consent/require-consent/index.md

```diff
@@ -7,7 +7,7 @@ Use the following steps to display the user consent dialog box as part of an Ope
 
 1. In the Admin Console, go to **Applications** > **Applications**.
 1. Select the OpenID Connect app that you want to require user consent for.
```

Comment:
From Ian's doc of the new App Wizard, replace this step with these two steps:
1. On the Create a new app integration page, select **OpenID Connect** in the **Sign-in method** section.
2. Choose either **Web Application** or **Single-Page Application** as the **Application type** for your integration.